linode/manager
# upcoming: [M3-7981] - Show the selected regions as chips in the AccessKeyDrawer

## Description 📝
Renders the selected regions as chips in the AccessKeyDrawer component.
The buckets table will be updated once region selection is completed.

## Preview 📷 
| Before | After |
| --- | --- |
| ![Screenshot 2024-04-18 at 10 20 02 AM](https://github.com/linode/manager/assets/114685994/fd11d779-1ccb-413f-b078-6f880d8b95cb) ![Screenshot 2024-04-18 at 10 20 18 AM](https://github.com/linode/manager/assets/114685994/88ea3808-2b4e-4089-a3e6-eb41d8ee5924) | ![Screenshot 2024-04-18 at 9 10 45 AM](https://github.com/linode/manager/assets/114685994/0c61eaac-da5a-4401-8152-795348217c63) ![Screenshot 2024-04-18 at 9 10 50 AM](https://github.com/linode/manager/assets/114685994/afa69f9d-eadb-4575-a784-99a2cc8092a7) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Turn on OBJ feature flag and MSW

### Verification steps
(How to verify changes)
- Navigate to http://localhost:3000/object-storage/access-keys
- Click on "Create Access Key"
- Select the multiple regions. 
- Verify the selected regions are rendered as chips.
- Verify the buckets table will be updated once regions selection is complete.  
- Verify there is no regression in OBJ Create / Edit Access flow. 

## As an Author I have considered 🤔

*Check all that apply*
